R/get_taxonlijsten_items.R

```diff
@@ -0,0 +1,152 @@
+#' @title Query to extract the taxa on a taxonlist from D0156_00_Taxonlijsten
+#'
+#' @description This function queries D0156_00_Taxonlijsten and gives an
```

Comment:
```suggestion
#' @description This function queries `D0156_00_Taxonlijsten` and gives an
```
Just a suggestion: putting variable names (or in this case a database name) between backticks makes it appear in another font style in the help in RStudio, and with a grey background on the website.

Reply:
suggestion applied for all functions